platform/src/main/java/org/stellar/anchor/platform/controller/Sep12Controller.java

```diff
@@ -61,6 +70,8 @@ public Sep12GetCustomerResponse getCustomer(
       method = {RequestMethod.PUT})
   public Sep12PutCustomerResponse putCustomer(
       HttpServletRequest request, @RequestBody Sep12PutCustomerRequest putCustomerRequest) {
+    debug(
```

Comment:
If this is repetitive, we may consider use filters to log request data in a separate PR. 

Here is a good example of Spring HTTP logging.